### ISSUE TITLE
The weights of YOLOX on the SportsMOT training dataset

### ISSUE THREAD

**Issue by hanhan0403** [hsiangwei0903/Deep-EIoU#24]
Your work on this project is truly remarkable. I'm really impressed by the effort and innovation you've put in. I'm interested in the weights of the detection network only on the SportsMOT training dataset. Did you train them yourself, or are you using the ones provided by MixSORT? If you trained them yourself, would it be possible for you to share these weights?

**Comment by hsiangwei0903** [comment]
The YOLOX weight is already provided in the repo ([here](https://github.com/hsiangwei0903/Deep-EIoU#1-model-preparation-for-demo-on-custom-dataset)). This model is trained on the SportsMOT training and validation set.
